### PR TITLE
Add alias name convention to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

This PR adds a convention for when to separate words in aliases by an underscore to CONTRIBUTING.md. Originally suggested by @weiji14 in https://github.com/GenericMappingTools/pygmt/pull/1190#issuecomment-814451969.

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -303,6 +303,12 @@ warn us and you can make a new commit with the formatted code.
 Even better, you can just write `/format` in the first line of any comment in a
 Pull Request to lint the code automatically.
 
+When wrapping a new alias, use an underscore to separate words bridged by vowels
+(aeiou), such as `no_skip` and `z_only`. Do not use an underscore to separate
+words bridged by only consonants, such as `distcalc`, and `crossprofile`. This
```
Comment (weiji14) [inline CONTRIBUTING.md:308]:
```suggestion
words bridged only by consonants, such as `distcalc`, and `crossprofile`. This
```

**Review by weiji14 — COMMENTED** [review]
Just one comment, otherwise looks good!

**Review by weiji14 — APPROVED** [review]

**Review by seisman — APPROVED** [review]

**Review by michaelgrund — APPROVED** [review]

**Review by willschlitzer — APPROVED** [review]
Looks good!

"Is 'aeiou' the term to describe when words are bridged by vowels?" -Me, just now 🤦‍♂️

**Comment by meghanrjones** [comment]
Sorry for forgetting to shorten the commit message when merging...

**Pull request merged**